GEOSldas_App/util/postproc/ObsFcstAna_stats/helper/util.py

```diff
@@ -0,0 +1,29 @@
+import os
```

Comment:
Move to ./util/shared/python if possible.  (When renaming the file, use "git mv oldname newname" so that git can keep track of the renaming.)